src/routes/blogs.rs

```diff
@@ -19,6 +19,38 @@ use plume_models::{
 use routes::{errors::ErrorPage, Page, PlumeRocket};
 use template_utils::Ructe;
 
+#[get("/?<page>", rank = 2)]
+pub fn custom_details(
```

Comment:
many routes will need duplicated functions, and reordering

in all those cases, we should replace the actual guts of the function with a private one, and all it, once the parameters have been processed successfully